packages/core/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "es2022",
     "declaration": true,
     "declarationDir": "./types",
-    "emitDeclarationOnly": true,
+    "emitDeclarationOnly": false,
```

Comment:
also, we're using babel, so we probably want this false. 

We use `babel` w the `typescript` plugin for generating our TS. Then we use `tsc` with `emitDeclarationOnly` to create the `typings`

Reply:
ok